### ISSUE TITLE
Alert type have no `updatedAt`  element in client/client.go 

### ISSUE THREAD

**Issue by yylt** [prometheus/alertmanager#2000]
**What did you do?**

use client-lib to fetch alerts, and filter alert which is  resolved or actived
```
import alertapi "github.com/prometheus/alertmanager/client"
...
tctx, canfn := context.WithTimeout(a.ctx, time.Second*10)
acalerts, _ = alertapi.NewAlertAPI(a.alertcli).List(tctx, "", "", true, true, true, true)
canfn()

for _,ale:=range acalerts{
    if ale.EndsAt.After(ale.UpdatedAt.Add(time.Minute *6)){
	 fmt.Println("actived alert",ale)
    }else{
	fmt.Println("resolved alert",ale)
    }
}

...
```

**What did you expect to see?**

complied success

**What did you see instead? Under which circumstances?**

there is not `UpdatedAt` element.

**Environment**

-

* Alertmanager version:

alertmanager 0.16.0

* Prometheus version:

	insert output of `prometheus -version` here (if relevant to the issue)

* Alertmanager configuration file:
```
global:
  resolve_timeout: 5m
route:
  receiver: "null"
  group_by:
  - job
  routes:
  - receiver: "null"
    match:
      alertname: Watchdog
  group_wait: 30s
  group_interval: 5m
  repeat_interval: 12h
receivers:
- name: "null"
templates: []
```

* Prometheus configuration file:
--

* Logs:
--

**Comment by simonpasquier** [comment]
I'm not sure what you're trying to do but `/api/v1/alerts` will only return firing alerts so `updatedAt` wont' tell you whether an alert is resolved or not.

There's also an API v2 available since AlertManager v0.17 (the API v1 is deprecated and should be removed in future versions). The v2 API has an [OpenAPI specification](https://github.com/prometheus/alertmanager/blob/master/api/v2/openapi.yaml) from which we generate server and [client](https://github.com/prometheus/alertmanager/tree/master/api/v2/client) code.

**Comment by yylt** [comment]
@simonpasquier  thanks, i just want use API to satify the demand , but I relize the webhook is wright way.
